packages/block-library/src/loginout/index.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Server-side rendering of the `core/loginout` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/loginout` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the login-out link or form.
+ */
+function render_block_core_loginout( $attributes ) {
+
+	// Build the redirect URL.
+	$redirect_url = isset( $attributes['redirectToCurrent'] ) && $attributes['redirectToCurrent']
+		? get_permalink( get_the_ID() )
+		: '';
+
+	// If logged-out and displayLoginAsForm is true, show the login form.
+	if ( ! is_user_logged_in() && ! empty( $attributes['displayLoginAsForm'] ) ) {
```

Comment:
Both the form and the link are missing the `wp-block-loginout` CSS class and the additional CSS class (from the option) on the front.

Reply:
Good point. Fixed.